src/content-scripts/components/ChatDialog/ChatDialog.ts

```diff
@@ -0,0 +1,65 @@
+import { createHtmlElement } from "../../../utils/createHtmlElement";
+import openSaucedLogoIcon from "../../../assets/open-sauced-orange-bg-logo.svg";
+
+export const ChatDialog = (ownerName: string, repoName: string) => {
```

Comment:
The `createHtmlElement` is a handy utility, but I wonder if we can use React instead here? We can insert a DOM element to be the root and mount the chat in there.

Also, I'd advise against using an inline onclick, `onclick: () => handleChatDialogClick(ownerName, repoName),`. It's not like an `onClick` in JSX. Prefer `addEventListener` instead.

Reply:
Yeah, the deeper I got into this, I realised react might be a better option here. I'll migrate, shouldn't be that hard this early. Thanks for your inputs!